package.json

```diff
@@ -13,8 +13,8 @@
     "autobind-decorator": "2.1.0",
     "connected-react-router": "4.3.0",
     "enzyme-adapter-react-16": "1.1.1",
+    "fetch-mock": "^6.5.0",
```

Comment:
nit: probably pin this like everything else.

Reply:
With yarn it doesn't really matter, since the lock file pins exact versions. Probably good to be consistent, but maybe we should consider using semver in this file for everything in the future?